cmd/artifact/main.go

```diff
@@ -0,0 +1,341 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+	artifact "github.com/taskcluster/taskcluster-lib-artifact-go"
+	"github.com/urfave/cli"
+)
+
+const (
+	ErrInternal = 1
+	ErrBadUsage = 2
+	ErrCorrupt  = 13
```

Comment:
`man sysexits` might offer some alternative suggestions for values here - although not required, might be nice.

Reply:
good thought!  I can't seem to load that manpage on Fedora, but the header is easy enough to understand.  0fcffd0